src/storage.ts

```diff
@@ -484,6 +484,10 @@ export class Storage extends Service {
    *     error should be retried and false otherwise.
    * @property {string} [userAgent] The value to be prepended to the User-Agent
    *     header in API requests.
+   * @property {object} [authClient] GoogleAuth client to reuse instead of creating a new one.
+   * @property {number} [timeout] The amount of time in milliseconds to wait for a http request before timing out.
```

Comment:
Is this the total timeout (covering all potential retries in a library call) or per-request?

Reply:
No, this is different.  `totalTimeout` is in `retryOptions`.

Reply:
Per request, I'll add that caveat to the sentence.